### PR TITLE
[RFR] Adding css guide updates from Monica + Margaret.

### PR DESCRIPTION
**Pull request by mgribouski**
Updating the 2 year old css guide with more guidelines & resources. Credit also goes to @margonaut.

### CODE REVIEW

File: webservices/css/styling_guidelines.md [hunk 1]
```diff
@@ -1,3 +1,31 @@
+# CSS Style Guide
+
+####The basics:
```
Comment (bfauble) [inline webservices/css/styling_guidelines.md]: headers need a space between the `#` and the header now to format properly

File: webservices/css/styling_guidelines.md [hunk 2]
```diff
@@ -43,3 +71,77 @@ It's better to have these all defined in one place, where they can be easily ref
 #### Favor targeting elements rather than pages
 
 Something in suspenders appends some seemingly helpful classes to the `<body>` element that specify the controller and action, but they are a trap. Use separate descriptive class names instead of having a single class that is styled or positioned differently depending on what template was rendered.
+
+####Images as Background*
+When adding images to your design, especially if it's going to be responsive, use a <div> tag with the background CSS property instead of <img> elements.
```
Comment (dgrosenblatt) [inline webservices/css/styling_guidelines.md]: 👍 👍 👍 👍 👍

File: webservices/css/styling_guidelines.md [hunk 3]
```diff
@@ -43,3 +71,77 @@ It's better to have these all defined in one place, where they can be easily ref
...
+Details in the design or less important components can be marked with a single-line comment.
+
+```
+/*   Footer Buttons   */
+.footer button { }
+
+.footer button:hover { }
+```
+
+Also, remember that CSS doesn't have single line  comments, so when commenting something out you still need to use the / / syntax.
```
Comment (bfauble) [inline webservices/css/styling_guidelines.md]: should this read "...so when commenting something out you still need to use the `/* */` syntax"?

File: webservices/css/styling_guidelines.md [hunk 4]
```diff
@@ -43,3 +71,77 @@ It's better to have these all defined in one place, where they can be easily ref
...
 
 Something in suspenders appends some seemingly helpful classes to the `<body>` element that specify the controller and action, but they are a trap. Use separate descriptive class names instead of having a single class that is styled or positioned differently depending on what template was rendered.
+
+####Images as Background*
+When adding images to your design, especially if it's going to be responsive, use a <div> tag with the background CSS property instead of <img> elements.
+
+This may seem like more work for nothing, but it actually makes it much easier to style images properly, keeping their original size and aspect-ratio, thanks to background-size, background-position, and other properties.
+
+CSS / HTML
+```
```
Comment (bfauble) [inline webservices/css/styling_guidelines.md]: you can add ` ```css ` or ` ```sass ` and add some color formatting to these codeblocks

File: webservices/css/styling_guidelines.md [hunk 5]
```diff
@@ -1,6 +1,8 @@
-# CSS Style Guide
+#
+CSS Style Guide
 
-####The basics:
+####
```
Comment (bfauble) [inline webservices/css/styling_guidelines.md]: sorry meant `#### The basics:` for the header formatting, not new line

**Review by dgrosenblatt — APPROVED** [review]

**Pull request merged**